### PR TITLE
Add pl.youkai.lanchat

### PR DESCRIPTION
**Pull request by youkai-pl**
Hello. I already published this project [here](https://github.com/flathub/io.github.tofudd.lanchat.terminal). I've moved my stuff to a new domain and git host. I want to republish the package under a new ID.

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "publish-delay-hours": 0
```
Comment (hfiguiere) [inline flathub.json:2]: this is not allowed.

File: pl.youkai.lanchat.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{
+  "app-id": "pl.youkai.lanchat",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "21.08",
```
Comment (hfiguiere) [inline pl.youkai.lanchat.json:4]: current runtime is 22.08

File: pl.youkai.lanchat.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.dotnet6"
+  ],
+  "finish-args": [
+    "--share=network",
+    "--env=DOTNET_ROOT=/app/dotnet/lib"
```
Comment (hfiguiere) [inline pl.youkai.lanchat.json:11]: nothing to display?

File: pl.youkai.lanchat.json [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        "cp -r --remove-destination /run/build/Lanchat/Lanchat.Terminal/bin/Release/net6.0/$RUNTIME/publish/* /app/bin/",
+        "install -D Lanchat.Terminal/Flatpak/pl.youkai.lanchat.svg /app/share/icons/hicolor/128x128/apps/pl.youkai.lanchat.svg",
+        "install -D Lanchat.Terminal/Flatpak/pl.youkai.lanchat.desktop /app/share/applications/pl.youkai.lanchat.desktop",
+        "install -D Lanchat.Terminal/Flatpak/pl.youkai.lanchat.metainfo.xml /app/share/metainfo/pl.youkai.lanchat.metainfo.xml"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://git.sr.ht/~youkai/lanchat",
+          "branch": "main"
```
Comment (hfiguiere) [inline pl.youkai.lanchat.json:54]: NEVER a branch. You need an actuall commit or tag.

File: pl.youkai.lanchat.json (hunk 3 above)
Comment (youkai-pl) [reply pl.youkai.lanchat.json:11]: It uses terminal UI.

**Comment by hfiguiere** [comment]
you will also need to EOL and redirect the original package.

https://github.com/flathub/flathub/wiki/App-Maintenance#end-of-life

**Comment by hfiguiere** [comment]
bot, build pl.youkai.lanchat

**Comment by youkai-pl** [comment]
bot, build pl.youkai.lanchat

**Comment by youkai-pl** [comment]
bot, build pl.youkai.lanchat

**Comment by youkai-pl** [comment]
bot, build pl.youkai.lanchat

**Comment by youkai-pl** [comment]
@hfiguiere I thinks it's ready now.

**Comment by barthalion** [comment]
/merge

Please remember to mark the old ID as end-of-life (+ rebase): https://github.com/flathub/flathub/wiki/App-Maintenance#end-of-life

**Pull request closed without merging**